napari/_vispy/layers/points.py

```diff
@@ -60,20 +57,23 @@ def _on_data_change(self):
 
         set_data = self.node._subvisuals[0].set_data
 
+        # use only last dimension to scale point sizes, see #5582
+        scale = self.layer.scale[-1]
```

Comment:
Optional: my intuitive pick for this would be the root of the volume scaling factor of the transform associated with the node. But this is better than nothing, should work for common cases, and I don't feel strongly here.

Reply:
> root of the volume scaling factor of the transform associated with the node

You mean for all dimensions? but yeah we discussed around this before, every option has some downside, so we settled on this for now and we'll see if people complain :P